models/prices/celo/prices_celo_tokens.sql

```diff
@@ -0,0 +1,27 @@
+{{ config(
+        schema='prices_celo',
+        alias = alias('tokens'),
+        materialized='table',
+        file_format = 'delta',
+        tags=['static','dunesql']
+        )
+}}
+SELECT 
+    token_id
+    , blockchain
+    , symbol
+    , contract_address
+    , decimals
+FROM
+(
+    VALUES
+
+    ('celo-celo', 'celo', 'CELO', 0x471ece3750da237f93b8e339c536989b8978a438, 24), 
```

Comment:
API: https://api.coinpaprika.com/v1/coins/celo-celo
SCAN: https://explorer.celo.org/mainnet/token/0x471EcE3750Da237f93B8E339c536989b8978a438/token-transfers

But the explorer shows its decimal is 18.
```suggestion
    ('celo-celo', 'celo', 'CELO', 0x471ece3750da237f93b8e339c536989b8978a438, 18), 
```

Reply:
yes you are correct. good catch.